spec/features/state_file/income_review_spec.rb

```diff
@@ -54,6 +56,7 @@ def advance_to_income_edit
     it "displays w2 info on edit screen" do
       advance_to_income_edit
 
+      expect(page).to be_axe_clean
```

Comment:
this passes even without the above change, but I wanted to add anyway while looking at this.